Scripts/CleanWorkspace.bat

```diff
@@ -1,5 +1,14 @@
 @echo off
 
 cd ..
-rmdir /s /q .\Built\
-rmdir /s /q .\Packages\Built\
\ No newline at end of file
+set RootDirectory=%cd%
+
+:: Remove C++ build output
+rmdir /s /q %RootDirectory%\Built\
+
+:: Remove MSFS Package output
+rmdir /s /q %RootDirectory%\Packages\Built\
+
+:: Clean EFB build output
+cd %RootDirectory%\Packages\PackageSources\meridian-simcache\EFB\SimCache
+call npm run clean:dist
```

Comment:
Do we want `npm run clean:dist` specifically?  I notice in the package.json there's also `clean:node_modules` (and `clean`, which does both dist _and_ node_modules)?

Is the idea that we don't want to clean node_modules since that appears on install rather than a build?

Reply:
Exactly that, I don't want this to put the workspace in a state where you then need to run Install.bat again. I was considering adding a `ResetWorkspace.bat` which deletes those modules as well.

Reply:
Ah, okay!  That makes sense.  I think `ResetWorkspace.bat` sounds like a reasonable idea, but that can come later too; no rush!